inst/report/Report_KRI.Rmd

```diff
@@ -74,17 +74,27 @@ if(setup$GroupLevel == "Site"){
 # subset data on latest snapshot date
 dfResults_current <- FilterByLatestSnapshotDate(params$dfResults)
 
-htmltools::div(
-  style = "table-layout: fixed; width: 100%; overflow: auto;",
-  Widget_GroupOverview(
-      dfResults = dfResults_current,
-      dfGroups = params$dfGroups,
-      dfMetrics = params$dfMetrics, 
-      strGroupLevel = setup$GroupLevel,
-      strGroupLabelKey = GroupLabelKey
-  )
+GroupOverViz <- Widget_GroupOverview(
+  dfResults = dfResults_current,
+  dfGroups = params$dfGroups,
+  dfMetrics = params$dfMetrics, 
+  strGroupLevel = setup$GroupLevel,
+  strGroupLabelKey = GroupLabelKey
+)
+
+# Remove sites whose metrics have no flags at all timepoints
+dfResults_flags <- FilterByFlags(params$dfResults)
+
+FlagOverViz <- Widget_FlagOverTime(
```

Comment:
Per issue #1653, we only want to include this chart when the number of unique snapshot dates in dfResults > 1, so adding some logic in `Report_KRI.Rmd` would help achieve that.